tests/installation/partitioning_lvm.pm

```diff
@@ -100,15 +100,15 @@ sub run {
             assert_screen "inst-select-root-disk";
             send_key 'alt-n';
         }
-        if (leap_version_at_least('15.0')) {
+        if (check_var('DISTRI', 'opensuse')) {
             send_key 'alt-a';
```

Comment:
I suggest to continue using approach we have used by defining keys in %cmd hash in [partitioning test module](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/partitioning.pm#L30) so we could put ``$cmd{enablelvm}  there and here simply ``send_key $cmd{enablelvm} unconditionally.

Reply:
+1